Sources/Datadog/Core/FeaturesConfiguration.swift

```diff
@@ -121,9 +121,10 @@ extension FeaturesConfiguration {
         if configuration.rumEnabled {
             var autoInstrumentation: RUM.AutoInstrumentation?
 
-            if let rumUIKitViewsPredicate = configuration.rumUIKitViewsPredicate {
+            if configuration.rumUIKitViewsPredicate != nil || configuration.rumUIKitActionsTrackingEnabled {
```

Comment:
isn't this confusing for the user? maybe we can use an enum here like 
`rumUIKitActionsTrackingEnabled = .forNone/.forSome(aPredicate)/.forAll`?

Reply:
The `predicate` is for RUM Views, whereas the `enabled` flag is for RUM Actions tracking. From the user perspective, it's very simple:
```swift
configuration
   .trackUIKitRUMViews(using: predicate) // to enable RUM Views tracking
   .trackUIKitActions(true) // to enable RUM Actions tracking
```

Reply:
ah okay my bad, i missed actions vs views